lib/src/subnav/subnav-right.directive.ts

```diff
@@ -0,0 +1,11 @@
+import { Directive, HostBinding } from '@angular/core';
+
+@Directive({
+    selector: '[hcSubnavRight]'
+})
+export class SubnavRightDirective {
+    @HostBinding('class.subnav-right')
```

Comment:
I've seen this pattern a few times, and it's totally fine, but just a thought - I think these four lines can be shoved into one. No need to scaffold out a getter function that just returns true.
```
     @HostBinding('class.subnav-right') true;
```

Reply:
Brilliant, I had no idea you could do that.  Change made.